packages/react-components/source/react/library/breadcrumb/Breadcrumb.md

```diff
@@ -26,6 +26,16 @@ import Link from '../link';
 </Breadcrumb>;
 ```
 
+## Back
+
+On occasion you will find your user will navigate directly from a high level view to one much further down the view workflow. At this point it may be more confusing to the user to show them the full breadcrumb trail through every screen as it is not the path they took. In this instance we recommend showing a "Back to" breadcrumb which will navigate the user back to the screen they we're just on. 
```

Comment:
I'm not sure how universal this recommendation should be... When we deep link in Relay, we go ahead and use the full breadcrumbs with a consistent IA. I know there are always exceptions (for example, we have one case where we do something like "Search results > X item" so the user can go back to the full search results instead of "X's parent > X item", but that's the exception.) This is a great option to have in Breadcrumb, but I'm not sure how much we should be encouraging folks to do this instead of trying the standard option first. What are you thoughts on rewriting this to be less of a "recommendation"?

Reply:
Went to UX (john murry) and got them to write the text they want displayed here 👍